Common/System/OSD.cpp

```diff
@@ -230,22 +235,37 @@ void OnScreenDisplay::SetProgressBar(std::string id, std::string &&message, int
 		}
 	}
 
+	if (message == "dorequest.php") {
+		found = found;
+	}
```

Comment:
Oops.

-[Unknown]

Reply:
That was just to have a place to set a breakpoint, left behind

Reply:
Yep, that's what it looked like.  Removed in #17782.

-[Unknown]